vocabularies/telephone-type.ttl

```diff
@@ -0,0 +1,41 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix sdo: <http://www.schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix tsl: <http://linked.data.gov.au/def/telephone-type/> .
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+
+<http://linked.data.gov.au/def/telephone-type> a owl:Ontology,
```

Comment:
How important is aligning the URI to the prefLabel?
If we keep it the same URI there is no need to recode the vocabulary references currently linked to this.
It's the lazy option, I'm happy to go either way, but i'd rather not create work elsewhere if i don't need to.

Reply:
> How important is aligning the URI to the prefLabel?

It's not required and sometimes can't be achieved. There can be no formal SKOS or other rule to mandate such a thing (you would need regular expression checking) but even if there was, it would defeat a SKOS purpose which is to separate identify (URI) from labels so that different situations can be catered for. 

I think its *required* that labels reflect the intention of the concept and *preferable* that they echo the URI. Sometimes you have numerical URIs (e.g. http://purlorg/obo/0_12345) that clearly can't have URIs echoing them.

Reply:
I'm happy to approve as-is and let the devs know of the redirection.